browser-test/src/checkbox.test.ts

```diff
@@ -50,6 +50,50 @@ describe('Checkbox question for applicant flow', () => {
       await logout(pageObject)
     })
 
+    afterEach(async () => {
+      await logout(pageObject)
+    })
+
+    it('Updates options in preview', async () => {
+      await loginAsAdmin(pageObject)
+
+      const adminQuestions = new AdminQuestions(pageObject)
+
+      await adminQuestions.createCheckboxQuestion(
+        {
+          questionName: '',
```

Comment:
question (blocking): why not set and verify question name?

Reply:
I didn't set it explicitly since we were never submitting the form and its value wouldn't show up in the preview. I added a value of `not-used-in-test` to hopefully make it more explicit. Updated the other two test files as well.